pkg/asset/internal/templates.go

```diff
@@ -539,12 +540,16 @@ spec:
     spec:
       containers:
       - name: etcd-operator
-        image: quay.io/coreos/etcd-operator:c391d8b7638deb81aa877773a0acce389f602415
+        image: quay.io/coreos/etcd-operator:dev
```

Comment:
Will this tag change at all? It'd be best to use a more specific release even if temporary.